### ISSUE TITLE
Proxy settings not honoured

### ISSUE THREAD

**Issue by ln5** [stom79/mastalab#399]
mastalab seems to be making outgoing connections to non-local addresses despite proxy being configured.

This was confirmed using application "Net Monitor", showing at least two connections not to 127.0.0.1. One of them is to hosted.fr1.maas.network.

Tested both with SOCKS and HTTP proxies, both provided by Orbot.

---

Instance: mastodon.social

Mastalab version: 1.8.5

Android version: 8.1.0 (CopperheadOS)

**Comment by stom79** [comment]
Only webviews are not set with the proxy currently. What are the queries?

**Comment by stom79** [comment]
I don't know how this app works, so I can't tell you. But hosted.fr1.maas.network sounds to be related to mastodon.social but queries should be rejected due to certificat issues. Are you seeing these calls when doing some special actions with the app like clicking on a link, watching an embedded media?

You are right for the translator, I need to upgrade the [library ](https://github.com/stom79/mytransl) too

**Comment by ln5** [comment]
> Are you seeing these calls when doing some special actions with the app like clicking on a link, watching an embedded media?

Trying to figure this out. Right now I'm not able to reproduce. One thing that I've changed since I saw outgoing connections to .network is unchecking "Use the built-in browser". Would that make sense?

**Comment by ln5** [comment]
My external browser uses Orbot as a proxy. Avoiding the built-in browser might be enough in order to not make outgoing connections, apart from the issue with translations.

**Comment by stom79** [comment]
Yes. It should be enough. I flagged this thread as an enhancement. I will add the proxy in the built-in browser and in the translation library.
